src/OrchardCore.Modules/OrchardCore.Redis/Startup.cs

```diff
@@ -31,13 +34,26 @@ public Startup(ShellSettings shellSettings, IShellConfiguration configuration, I
             _logger = logger;
         }
 
+        private static bool AlwaysValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+        {
+            // expect the certificate to exist, but not much else
+            return (sslPolicyErrors & SslPolicyErrors.RemoteCertificateNotAvailable) == 0;
+        }
+
         public override void ConfigureServices(IServiceCollection services)
         {
             try
             {
-                var configuration = _configuration["OrchardCore_Redis:Configuration"];
+                var section = _configuration.GetSection("OrchardCore_Redis");
+
+                var configuration = section["Configuration"];
                 var configurationOptions = ConfigurationOptions.Parse(configuration);
-                var instancePrefix = _configuration["OrchardCore_Redis:InstancePrefix"];
+                var instancePrefix = section["InstancePrefix"];
+
+                if (section.GetValue("DisableSSLCertificate", false))
```

Comment:
Why not `EnableSslCertificate`?

Reply:
Two reasons: 1) this is what MikeAlhayek suggested, and 2) certificate verification should be (and is) enabled by default. Disabling it could have security implications and users should only do so when necessary.

Reply:
I mean if's enabled by default

```json
    "EnableSSLCertificate": true
```

Reply:
I suppose it would be possible if the option defaults to true if it does not exist in the config file (e.g. because of an upgrade from a previous release). However, naming it like that seems it could be confusing for the user, because then it sounds a lot like an option for enabling SSL, not an option for allowing SSL certificate validation to be performed in the normal way. After all, this new option does not enable SSL, you'd have to put "ssl=true" in the "Configuration" option for that. This new option just disables the certificate checking that would otherwise normally happen when you put in "ssl=true". So, while it might be feasible to make this an option that defaults to true, I'm thinking it should probably have a clearer name if that's the way you really want it.

Reply:
I think `DisableSSLCertificate` is fine.

Reply:
@ovekaaven actually probably better `DisableCertificateVerification`

Reply:
Done, renamed to DisableCertificateVerification.